integration_tests/src/main/python/map_test.py

```diff
@@ -409,20 +417,31 @@ def test_get_map_value_element_at_map_string_col_keys(data_gen):
         conf={'spark.sql.ansi.enabled': False})
 
 
+@pytest.mark.skipif(is_spark_340_or_later(),
+                    reason="Only in Spark 3.1.X ~3.3.X + ANSI mode, map key throws on no such element")
```

Comment:
This is already handled a couple of lines below.  Only caught this because last night's 340 build passed, and I was wondering how it could have without this going in.  Maybe the headline should be changed to reflect that this isn't fixing failures but adding new tests?
```suggestion
```